pkg/windows/windows.go

```diff
@@ -379,6 +379,12 @@ func (vm *windows) Reinitialize() error {
 }
 
 func (vm *windows) EnsureRequiredServicesStopped() error {
+	// In this transitional period of migrating services to WICD's responsibility, WICD must be stopped first to ensure
+	// it does not start any of these stopped services
```

Comment:
So will this go away after the transition? If yes, then the story where this should be updated to call this out and the Jira issue linked here.